TrafficCapture/trafficCaptureProxyServer/README.md

```diff
@@ -0,0 +1,100 @@
+# Capture Proxy
+
+## How to attach a Capture Proxy on a coordinator node.
+
+Follow documentation for [deploying solution](../../deployment/README.md). Then, on a cluster with at least two coordinator nodes, the user can attach a Capture Proxy on a node by following these steps:
+Please note that this is one method for installing the Capture Proxy on a node, and that these steps may vary depending on your environment.
+
+
+These are the **prerequisites** to being able to attach the Capture Proxy:
+
+* **Make sure that the node and your MSK client are in the same VPC and Security Groups**
+    * Add the following IAM policy to the node/EC2 instance so that it’s able to store the captured traffic in Kafka:
+    * From the AWS Console, go to the EC2 instance page, click on **IAM Role**,  click on **Add permissions**, choose **Create inline policy**, click on **JSON VIEW** then add the following policy (replace region and account-id).
+
+    ```json
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Action": "kafka-cluster:Connect",
+                "Resource": "arn:aws:kafka:<region>:<account-id>:cluster/migration-msk-cluster-<stage>/*",
+                "Effect": "Allow"
+            },
+            {
+                "Action": [
+                    "kafka-cluster:CreateTopic",
+                    "kafka-cluster:DescribeTopic",
+                    "kafka-cluster:WriteData"
+                ],
+                "Resource": "arn:aws:kafka:<region>:<account-id>:topic/migration-msk-cluster-<stage>/*",
+                "Effect": "Allow"
+            }
+        ]
+    }
+  ```
+
+* From **linux command line** of that EC2 instance: Check that the **JAVA_HOME environment variable is set** properly `(echo $JAVA_HOME)`, if not, then try running the following command that might help set it correctly:
```

Comment:
This is not following the same pattern as the other steps. I would start with something like "Verify Java installation is  accessible."

Reply:
Addressed in new commit.